examples/framework/preferences/minimalEditor/js/minEditor.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one of these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+    /**
+     * Panel for the auto-pilot preference
+     */
+    fluid.defaults("minEditor.panels.autoPilot", {
+        gradeNames: ["fluid.prefs.panel"],
+
+        // the Preference Map maps the information in the primary schema to this panel
+        preferenceMap: {
+            // the key must match the name of the pref in the primary schema
+            "minEditor.autoPilot": {
+                // this key is the path into the panel's model where this preference is stored
+                "model.autoPilot": "default"
+            }
+        },
+
+        // selectors identify elements in the DOM that need to be accessed by the code;
+        // in this case, the Renderer will render data into these particular elements
+        selectors: {
+            autoPilot: ".mec-autoPilot"
+        },
+
+        // the ProtoTree is basically instructions to the Renderer
+        // the keys in the prototree match the selectors above
+        protoTree: {
+            // this value is an IoC reference to the last part of the model path in the preferenceMap
+            autoPilot: "${autoPilot}"
```

Comment:
Similar to the comment from the [FLUID-5757 pull request](https://github.com/fluid-project/infusion-docs/pull/75#discussion_r44304768), I'm not sure that `${autoPilot}` is an IoC reference.